src/sdk/analytics/platform/vtex.ts

```diff
@@ -1,5 +1,5 @@
 import type { AnalyticsEvent } from '@faststore/sdk'
 
 export default function sendEvent(event: AnalyticsEvent) {
-  window.sendrc(event.name, event.params)
+  window.sendrc?.(event.name, event.params)
```

Comment:
Is it related to the cypress?

Reply:
Isn't, but have broken when i changed the versions